tests/e2e/set-ffi-env-e2e

```diff
@@ -129,7 +129,7 @@ create_qm_disks() {
   mkdir /var/qm
   for disk in $disks_arr; do
      if [[ ${disk} == "vda" || \
-           $(echo "${disk_table}" | grep -c "${disk}" ) -eq 1 ]];then
+           $(echo "${disk_table}" | grep -c "${disk}" ) -eq 1 && ${disk} != "zram0" ]];then
```

Comment:
what's the trick with zram0?

Reply:
It was failing the setup script while running inside fedora machine